src/qt3utils/applications/qt3ple/main.py

```diff
@@ -0,0 +1,503 @@
+import argparse
+import importlib
+import importlib.resources
+import logging
+import pickle
+from threading import Thread
+
+import matplotlib
+from matplotlib.backends.backend_tkagg import FigureCanvasTkAgg, NavigationToolbar2Tk
+import matplotlib.pyplot as plt
+import nidaqmx
+import numpy as np
+import tkinter as tk
+import yaml
+
+from qt3utils.datagenerators import plescanner
+
+matplotlib.use('Agg')
+
+parser = argparse.ArgumentParser(description='NI DAQ (PCIx 6363) / PLE Scanner',
+                                 formatter_class=argparse.ArgumentDefaultsHelpFormatter)
+parser.add_argument('-q', '--quiet', action='store_true',
+                    help='When true,logger level will be set to warning. Otherwise, set to "info".')
+parser.add_argument('-cmap', metavar='<MPL color>', default='gray',
+                    help='Set the MatplotLib colormap scale')
+args = parser.parse_args()
+
+logger = logging.getLogger(__name__)
+logging.basicConfig()
+
+if args.quiet is False:
+    logger.setLevel(logging.INFO)
```

Comment:
Can you change this to something similar to what the qt3scan main file looks like today?

```python
parser = argparse.ArgumentParser(description='QT3Scan', formatter_class=argparse.ArgumentDefaultsHelpFormatter)
parser.add_argument('-v', '--verbose', type=int, default=2, help='0 = quiet, 1 = info, 2 = debug.')
args = parser.parse_args()

logger = logging.getLogger(__name__)
logging.basicConfig()

if args.verbose == 0:
    logger.setLevel(logging.WARNING)
if args.verbose == 1:
    logger.setLevel(logging.INFO)
if args.verbose == 2:
    logger.setLevel(logging.DEBUG)
```

Reply:
correct